pkgs/top-level/all-packages.nix

```
@@ -9280,6 +9280,8 @@ with pkgs;
   netdata = callPackage ../tools/system/netdata {
     inherit (darwin.apple_sdk.frameworks) CoreFoundation IOKit;
   };
+  # Exposed here so the bots can auto-upgrade it
+  netdata-go-d-plugin = callPackage ../tools/system/netdata/go.d.plugin.nix {};
```

I think we can use `netdata-go-plugins` since it's used by Alpine https://repology.org/project/netdata-go-plugins/versions
```suggestion
  netdata-go-plugins = callPackage ../tools/system/netdata/go.d.plugin.nix {};
```

---

:+1: thanks, done!